types/src/on_chain_config/execution_config.rs

```diff
@@ -70,7 +70,11 @@ impl OnChainExecutionConfig {
     /// Features that are ready for deployment can be enabled here.
     pub fn default_for_genesis() -> Self {
         OnChainExecutionConfig::V4(ExecutionConfigV4 {
-            transaction_shuffler_type: TransactionShufflerType::SenderAwareV2(32),
+            transaction_shuffler_type: TransactionShufflerType::Fairness {
+                sender_conflict_window_size: 256,
+                module_conflict_window_size: 1,
+                entry_fun_conflict_window_size: 2,
```

Comment:
I would increase this a bit - we have 8 threads, 6x speedup from parallelism. so I would set it in the 3-5 range (3 if you want less aggressive, 5 if you want more aggressive)

Reply:
`{32/256, 2, 3}`? The dominating entry point will still have parallelism of 2 in any case.